crates/topos/src/edge/mod.rs

```diff
@@ -99,8 +100,18 @@ impl CommandConfig {
 
         let (running_out, _) = tokio::join!(running, logging);
 
-        info!("The Edge process is terminated");
-        running_out
+        let exit_status = running_out?;
+
+        info!(
```

Comment:
That's a lot of information, let's keep that like that and see later if it doesn't display too much things.